src/wwwroot/controllers/TemplatesCtrl.js

```diff
@@ -50,6 +51,10 @@
         });
     }
 
+    function previewTemplate(thumbnailUrl) {
+      window.open(thumbnailUrl);
```

Comment:
it is weird. Now you have the HTML rendering, why showing a small image?

Reply:
This shouldn't be here. My bad!